src/client.rs

```diff
@@ -13,7 +13,7 @@ pub use signatures::{SignatureError, Verification};
 use crate::client::signatures::openssl_verifier::OpenSSLVerifier as DefaultVerifier;
 
 #[cfg(not(feature = "openssl_verifier"))]
-use crate::client::signatures::dummy_verifier::DummyVerifier as DefaultVerifier;
+use crate::client::signatures::default_verifier::DefaultVerifier as DefaultVerifier;
```

Comment:
I'm not a huge fan of having the `DefaultVerifier` not say that it does not, in fact, verify -- that's why I suggested the awkward name `AlwaysAcceptsVerifier` or whatever it was.

I don't feel strongly about this; if you and your other reviewers are good with `DefaultVerifier`, roll on.